### DISCUSSION TITLE
Responses not shown: 1999 form/Massena

### DISCUSSION THREAD

**Discussion by tamihuber** [paulwinterbauer/NBIC#100] [Q&A] [answered]
http://serc4.si.edu:8001/NBIC/review/2826518/#

(Also- separately, I left a question in discussion tab-- did I do it correctly?)

**Comment by paulwinterbauer** [comment, marked as answer]
Not a bug, we don't have responses produced for 1999 forms.  In any case, this was received from Massena right?  Would it make sense to send them responses?

**Comment by tamihuber** [comment]
Right! This is so strange, because it's a massena, but one of those rare massenas that doesn't list Montreal/st L river as the last location. Not a bug, in any case. Thx!

**Comment by paulwinterbauer** [comment]
This kind of question would be a good candidate for discussions.  If the question leads to some work to be done, at that point we can add an issue.  Otherwise, especially if we generalize this question, it becomes part of a knowledge base for why some responses aren't produced that we can reference later.

**Comment by tamihuber** [comment]
Ah, I dont think I would ever have put this as a discussion initially--because I didn't realize the massena thing.
